sdk/storage/Azure.Storage.DataMovement.Files.Shares/src/ShareFileDestinationCheckpointData.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Azure.Storage.DataMovement.Files.Shares
 {
-    internal class ShareFileDestinationCheckpointData : StorageResourceCheckpointData
+    internal class ShareFileDestinationCheckpointData : StorageResourceCheckpointDataInternal
```

Comment:
Not for this PR - but potentially in the future we probably have to do this for Blobs, once we move the tests into the datamovement blobs test package